vtex-apps/store-graphql
# Remove profile picture upload and related methods that are now useless

#### What problem is this solving?
It removes the file upload vulnerability.

#### How should this be manually tested?
- Login into [cobasi](https://bianca--cobasi.myvtex.com/) or [storecomponents](https://bianca--storecomponents.myvtex.com/);
- Go into your account;
- Try to change your profile picture (the add picture button next to the profile picture still appears but is going to be removed - https://github.com/vtex-apps/my-account/pull/145 and https://github.com/vtex-apps/my-account/pull/146);
- The picture shouldn't change and there shouldn't be any errors.

You can also try the actual mutations:
- [Cobasi](https://bianca--cobasi.myvtex.com/_v/private/vtex.store-graphql@2.113.5/graphiql/v1?operationName=UpdateProfilePicture&query=mutation%20UpdateProfilePicture(%24file%3A%20Upload!)%20%7B%0A%20%20updateProfilePicture(file%3A%20%24file)%20%7B%0A%20%20%20%20cacheId%0A%20%20%20%20profilePicture%0A%20%20%20%20addresses%7B%0A%20%20%20%20%20%20state%0A%20%20%20%20%7D%0A%20%20%20%20corporateName%0A%20%20%7D%0A%7D&variables=%7B%0A%20%20%22file%22%3A%20%22%22%0A%7D)
- [StoreComponents](https://bianca--storecomponents.myvtex.com/_v/private/vtex.my-orders-graphql@0.2.0/graphiql/v1?operationName=UpdateProfilePicture&query=mutation%20UpdateProfilePicture(%24file%3A%20Upload!)%20%7B%0A%20%20updateProfilePicture(file%3A%20%24file)%20%7B%0A%20%20%20%20cacheId%0A%20%20%20%20profilePicture%0A%20%20%20%20addresses%7B%0A%20%20%20%20%20%20state%0A%20%20%20%20%7D%0A%20%20%20%20corporateName%0A%20%20%7D%0A%7D&variables=%7B%0A%20%20%22file%22%3A%20%22%22%0A%7D)

#### Checklist/Reminders


#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
✔️ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes

Related to:
https://github.com/vtex-apps/my-account/pull/145
https://github.com/vtex-apps/my-account/pull/146
